### PR TITLE
✨ Add groupCard component

### LINKED ISSUE #6: Component - Group Card
### Screenshot
image

### User Story
As a user, 
I want to see the main information of the sport groups,
so I can decide which groups are interesting to explore further. 

### To Do
- [ ] Link Group Card to Group Details Page
- [ ] Map Group Card components
- [ ] `Component` Group image
    - [ ] Add group image
    - [ ] layer with linear gradient over group image
- [ ] group name
- [ ] location text
- [ ] `Component`date text
- [ ] `Component`time
    - [ ] time text
    - [ ] time icon
- [ ] `Component`distance
    - [ ] distance text
    - [ ] distance icon
- [ ] `Component` sports icon

### Acceptance Criteria

- Given I am on the Group List page, then I see these fields of the group card: group image, group name, location, sports icon, date, time and distance.

- Given I click on one of the group cards in the list, then I am directed to the group details page.

### PR DESCRIPTION
**Pull request by Dev-dfm**
Add groupCard component

Try it out:
https://fit-2-gether-dev-dfm-is-azjyb2.herokuapp.com/storybook/?path=/story/component-card--group-card

fixes #6

### CODE REVIEW

**Review by dhnazarenko — COMMENTED** [review]
looks really nice! 🚀

File: src/app/components/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  location,
+  month,
+  date,
+  time,
+  distance,
+}: CardProps): JSX.Element {
+  return (
+    <div className={styles.card}>
+      <div className={styles.imageFrame}>
+        {/* <img className={styles.card__image} src="/boxing.jpeg" /> */}
```
Comment (lajanzen) [inline src/app/components/Card/Card.tsx:27]: is this comment necessary?

File: src/app/components/Icons/BoxingIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="22.491"
+      height="30"
+      viewBox="0 0 22.491 30"
+    >
+      <g transform="translate(3524.019 -4369.523)">
+        <path
+          d="M-3513.8,4370.662h0a.636.636,0,0,1,.269.069,17.423,17.423,0,0,1,4.639,1.174,4.762,4.762,0,0,1,2.277,2.042,5.271,5.271,0,0,1,.835,2.968l-.1,1.845a.626.626,0,0,1-.608.591c.564,0,1.8-.023,1.8-.023a2.515,2.515,0,0,1,2.538,2.509v.405a2.441,2.441,0,0,1-.747,1.757l-2.675,2.584v.32a5,5,0,0,1-5,5h-7.822a5,5,0,0,1-5-5v-9.818a5.613,5.613,0,0,1,1.04-3.07,5.127,5.127,0,0,1,2.482-2.048,20.528,20.528,0,0,1,6.072-1.308Z"
+          fill="#88ebe8"
```
Comment (lajanzen) [inline src/app/components/Icons/BoxingIcon.tsx:14]: do you maybe have a color variable for this color?

**Review by lajanzen — APPROVED** [review]

File: src/app/components/Card/Card.module.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  margin: 0 auto;
```
Comment (phlgr) [inline src/app/components/Card/Card.module.css:3]: I would prefer not setting a margin here but rather the parent centering the element

File: src/app/components/Card/Card.tsx (hunk 1 above)
Comment (Dev-dfm) [reply src/app/components/Card/Card.tsx:27]: Nope. Thanks 👍

**Pull request merged**